twilight-model/src/channel/mod.rs

```diff
@@ -111,6 +111,9 @@ pub struct Channel {
     /// ID of the last message pinned in the channel.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub last_pin_timestamp: Option<Timestamp>,
+    /// For group DM channels: whether the channel is managed by an application via the `gdm.join` OAuth2 scope.
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Discord's documentation often has an odd tone to it, either because fields are limited to one line or because types tend to have asterisks*****

Maybe this would read more gooder with something like this? Plus we get intradoc links!
```suggestion
    /// Whether the channel is managed by an application via the [`GDM_JOIN`]
    /// scope.
    ///
    /// This is only applicable to [group channels].
    ///
    /// [`GDM_JOIN`]: crate::scope::GDM_JOIN
    /// [group channels]: ChannelType::Group
    #[serde(skip_serializing_if = "Option::is_none")]
```